.zipignore

```diff
@@ -0,0 +1,47 @@
+# A set of files to remove from npm run build:zip
+.distignore
+.editorconfig
+.git/*
+.gitignore
+.gitlab-ci.yml
+.travis.yml
+.DS_Store
+.zipignore
+Thumbs.db
+behat.yml
+bin/*
+circle.yml
+composer.json
+composer.lock
+Gruntfile.js
+package.json
+package-lock.json
+phpunit.xml
+phpunit.xml.dist
+multisite.xml
+multisite.xml.dist
+phpcs.xml
+phpcs.xml.dist
+README.md
+wp-cli.local.yml
+yarn.lock
+tests/*
+config/*
+node_modules/*
+vendor/bin/*
+vendor/dealerdirect/*
+vendor/doctrine/*
+vendor/phar-io/*
+vendor/phpcompatibility/*
+vendor/phpdocumentor/*
+vendor/phpspec/*
+vendor/phpunit/*
+vendor/sebastian/*
+vendor/squizlabs/*
+vendor/theseer/*
+vendor/webmozart/*
+vendor/woocommerce/*
+vendor/wp-coding-standards/*
+*.sql
+*.tar.gz
+*.zip
```

Comment:
Isn't this just a copy of `.distignore`? The release builds are explicit in what they include.

https://github.com/woocommerce/woocommerce-admin/blob/c8eb1e44c820192dafbf2e0361f9fabfe591affb/bin/github-deploy.sh#L121-L132

Do you think it would be good to reuse that logic?

Reply:
> Isn't this just a copy of .distignore?

I started with .distignore. The zip command needs the `/\*` on the end of folder names to exclude folders.

Reply:
@psealock We could move that zip command to a separate .sh and call it from both. Sound good?

Reply:
@psealock Thanks for the suggestion. The PR has been updated to re-use the deploy zip command. I added the exclusion list to avoid running `composer install --no-dev` to remove the dev packages before `zip`ping.